bundle/edu.gemini.spdb.reports.collection/src/main/scala/edu/gemini/dbTools/timingwindowcheck/TimingWindowCheckMailer.scala

```diff
@@ -0,0 +1,35 @@
+package edu.gemini.dbTools.timingwindowcheck
+
+import edu.gemini.dbTools.mailer.{ Mailer, MailerType, ProgramAddresses }
+import edu.gemini.spModel.core.{ Site, SPProgramID }
+
+import scalaz._
+import Scalaz._
+import scalaz.effect.IO
+
+sealed abstract class TimingWindowCheckMailer(mailer: Mailer) {
+
+  def notifyPendingCheck(
+    pid:   SPProgramID,
```

Comment:
When it comes time to write the body of the email, we'll also want to know the list of observations whose window closing is coming up.